presto-main/src/main/java/com/facebook/presto/sql/planner/DomainTranslator.java

```diff
@@ -690,7 +690,7 @@ private int compareOriginalValueToCoerced(Type originalValueType, Object origina
         @Override
```

Comment:
Commit message is too long, it wraps in Github. See https://chris.beams.io/posts/git-commit/. Also please add the explanation from PR to the commit message.